params/protocol_params.go

```diff
@@ -179,8 +179,6 @@ const (
 	BlobTxPointEvaluationPrecompileGas = 50000   // Gas price for the point evaluation precompile.
 	BlobBaseCost                       = 1 << 13 // Base execution gas cost for a blob.
 
-	HistoryServeWindow = 8192 // Number of blocks to serve historical block hashes for, EIP-2935.
```

Comment:
I agree the value is incorrect, but this is still a protocol param and so it should stay defined here.

Reply:
Friendly ping @fjl

Reply:
We can't really configure it differently, since it's an internal value in the contract.

Reply:
The value is defined by https://eips.ethereum.org/EIPS/eip-2935, and changing it would change the contract code and its address. So I think it's fair to say, it's not a tunable parameter.

Reply:
I think having `HistoryServeWindow = 8191` in `protocol_params.go` is nice for being explicit about this, even if it is not used in geth (or only used in tests).